modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/utils/getFields.ts

```diff
@@ -146,11 +144,13 @@ function getValidFields({
 	return fields;
 }
 
-export default async function getFields(fdsView: FDSViewType) {
-	const {restApplication, restSchema} = fdsView[
-		OBJECT_RELATIONSHIP.FDS_ENTRY_FDS_VIEW
-	];
-
+export default async function getFields({
+	restApplication,
+	restSchema,
+}: {
+	restApplication: string;
+	restSchema: string;
+}) {
```

Comment:
@chestofwonders you were also looking to do this, so I fit solution to this API. My reason for ti was different, destructuring was on different variable based on FF.